cloudinit/config/cc_rsyslog.py

```diff
@@ -246,10 +246,7 @@ def __init__(
         self.proto = proto
 
         self.addr = addr
-        if port:
-            self.port = int(port)
-        else:
-            self.port = None
+        self.port = int(port) if port is not None else None
```

Comment:
For what it's worth, the first assignment could just be annotated with `Optional[int]`, but this is fine too.